compiler/ballerina-backend-jvm/src/main/ballerina/compiler_backend_jvm/jvm_constants.bal

```diff
@@ -191,9 +191,10 @@ final int V1_8 = 0 << 16 | 52;
 // jvm runtime values related classes
 const string MAP_VALUE = "org/ballerinalang/jvm/values/MapValue";
 const string ARRAY_VALUE = "org/ballerinalang/jvm/values/ArrayValue";
+const string OBJECT_VALUE = "org/ballerinalang/jvm/values/ObjectValue";
```

Comment:
Shall we use some consistence naming convention for the constants here? May be we an use one pattern for jvm classes and one or ballerina types, values ?

Reply:
The current convention is as follows:

- Values: \<typename\>_TYPE 
- Type names: \<typename\>_VALUE 
- Other JVM classes: same name as their java class name

There are some cases that belongs to two of these categories. eg `java.lang.Object`. Those were named according to the 3rd category.

Reply:
ack, if we have a identifiable pattern